gyg/style.css

```diff
@@ -186,4 +186,25 @@ textarea {
 
 .footer-sm-menu>li {
     list-style: none;
+}
+
+@media screen and (max-width: 480px) {
```

Comment:
Mobile first approach means that you define the styles for mobile in a first step and after, you adapt them from other devices. You use to define the mobile styles without a @media sentence but after you can use something like:

```css
@media screen and (min-width: 481px) { ...
```